<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="cpm10155.local" ignored="0" name="hub spot regression Test" tests="4" failures="1" timestamp="16 Jan 2020 14:04:01 GMT" time="28.861" errors="0">
  <testcase name="verifyLoginPageTitleTest" time="5.007" classname="com.qa.hubspot.tests.LoginTests">
    <failure type="java.lang.AssertionError" message="expected [HubSpot Login123] but found [HubSpot Login]">
      <![CDATA[java.lang.AssertionError: expected [HubSpot Login123] but found [HubSpot Login]
at com.qa.hubspot.tests.LoginTests.verifyLoginPageTitleTest(LoginTests.java:32)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- verifyLoginPageTitleTest -->
  <testcase name="verifySignUpLinkTest" time="4.467" classname="com.qa.hubspot.tests.LoginTests"/>
  <testcase name="loginCorrectCredentialsTest" time="5.44" classname="com.qa.hubspot.tests.LoginTests"/>
  <testcase name="testSuccessPercentage" time="4.946" classname="com.qa.hubspot.tests.LoginTests"/>
</testsuite> <!-- hub spot regression Test -->
